apis/core/hydra/index.ttl

```diff
@@ -0,0 +1,15 @@
+@prefix hydra: <http://www.w3.org/ns/hydra/core#> .
+@prefix code:  <https://code.described.at/> .
+
+<api> a                    hydra:ApiDocumentation ;
+      hydra:entrypoint     <> ;
+      hydra:supportedClass hydra:Resource,
+                           hydra:Collection .
+
+hydra:Resource hydra:supportedOperation [ hydra:method       "GET" ;
+                                          code:implementedBy [ a         code:EcmaScript ;
+                                                               code:link <node:@hydrofoil/labyrinth/resource#get> ] ] .
```

Comment:
Hydra crash course 💥

This is how `hydra-box` attaches actual `express` handlers to RDF classes, using the `code:implementedBy` property.

In the simple case reading resources, generic handlers are loaded from `labyrinth` to provide the `GET` "routing" for most resources. This should cater for 90% of the read requests.